cpp/build-support/build-lz4-lib.sh

```diff
@@ -17,9 +17,14 @@
 # specific language governing permissions and limitations
 # under the License.
 #
+MAKE="make"
+if [ -n "$MSYSTEM" ]; then
+  MAKE="mingw32-make"
+fi
```

Comment:
How about setting correct `PATH` such as `PATH=/mingw32/bin:$PATH` by user instead of this?

Reply:
I found that the mingw-w64-x86_64-make package doesn't install `make` but rather `mingw32-make`, unfortunately. (And from what I understand, the msys2 make package shouldn't be mixed with the mingw toolchain we use here.)

Reply:
Oh, I see.
I confirmed it at: https://packages.msys2.org/package/mingw-w64-x86_64-make?repo=mingw64